pkg/types/ovirt/machinepool.go

```diff
@@ -29,6 +29,17 @@ type MachinePool struct {
 	// The affinity groups should exist on the oVirt cluster or created by the OpenShift installer.
 	// +optional
 	AffinityGroupsNames []string `json:"affinityGroupsNames,omitempty"`
+
+	// AutoPinningPolicy defines the policy to automatically set the CPU
+	// and NUMA including pinning to the host for the instance.
+	// +kubebuilder:validation:Enum="";disabled;existing;adjust
+	// +optional
+	AutoPinningPolicy string `json:"autoPinningPolicy,omitempty"`
```

Comment:
I prefer not to. It made sense before the current change. We are going to block `existing` policy in RHV.
The difference was that for VM type `high performance` if nothing specified(""), then `existing` policy will auto-set. A patch on terraform will be posted to remove this behavior, but until then, this is the difference. In the near future, we won't have any.

Reply:
I am confused. You prefer not to tell the user what the behavior is going to be. And then you are also planning to change the behavior later. Both of those are troublesome to me.

Reply:
Currently with RHV version <= 4.6 when you create a VM with High Performance you get a VM with existing Auto Pinning Policy, on RHV >= 4.7 that will change and you will get it with "disabled" and the "existing" policy will be blocked (there are some major problems with it) - This is all RHV specific stuff that OCP shouldn't care about.

Currently I think that there is no difference between "" and "disabled" from openshift eyes.
On masters "disabled" is added by the terraform code when nothing is set https://github.com/openshift/installer/pull/4873/files#diff-89b21506ff273f62d250d753ad7a41d42f7ca10249cdcc4b2a04ebfb65032a36R118

On workers the machine logic will create it with disabled if nothing is specified 

@liranr23 correct me if I'm wrong

Maybe we should add a default like we do with high performance mode here: https://github.com/openshift/installer/blob/master/pkg/asset/machines/worker.go#L122-L134

Reply:
If the "existing" policy will be blocked, how will that affect clusters installed using the "existing" policy? Should OpenShift be accepting "existing" as a valid policy now?

Reply:
Nope, there are no clusters that are installed with existing policy.
This is a new thing in Openshift 4.8, before 4.8 we didn't have any Auto Pinning Policy logic

Reply:
That's correct. In RHV UI, the existing policy will take place on VMs set as High Performance. In terraform we mimic this behavior, which going to be dropped both RHV and terraform.
The existing also going to be blocked until we will change the behavior in RHV(blocked since 4.7). Currently this behavior is somewhat behind the scenes, and going to drop anyway.

The "existing" policy is dropped in the code in this PR as it's irrelevant and we shouldn't confuse users. This is not a valid option at the moment. Once we will have it back in RHV, we can add it to the installer.